packages/@angular/cli/tasks/lint.ts

```diff
@@ -60,6 +60,9 @@ export default Task.extend({
         errors += result.failureCount;
         results = results.concat(result.output.trim().concat('\n'));
       });
+      let allResults = results.split('\n');
+      allResults = allResults.filter((item, index) => item.indexOf(item) === index);
+      results = allResults.toString();
```

Comment:
Shouldn't the new line character be re-inserted into the string? So instead of just calling `toString` use `join('\n')`